pkg/api/customization/monitor/cluster_graph_action.go

```diff
@@ -69,7 +75,20 @@ func (h *ClusterGraphHandler) QuerySeriesAction(actionName string, action *types
 	var svcName, svcNamespace, svcPort, token string
 	if inputParser.Input.Filters["resourceType"] == "istiocluster" {
 		inputParser.Input.MetricParams["namespace"] = ".*"
-		svcName, svcNamespace, svcPort = monitorutil.IstioPrometheusEndpoint()
+
+		selector := fields.OneTermEqualSelector(systemProjectLabel, "true")
+		projects, err := h.projectsGetter.Projects(clusterName).List(metav1.ListOptions{LabelSelector: selector.String()})
```

Comment:
Can you use function `GetSystemProject` in `pkg/project/project.go`? That would be more stable way to get system project.

Reply:
updated.